### ISSUE TITLE
Pyright can't analyze the type of numpy.pi correctly

### ISSUE THREAD

**Issue by spindensity** [microsoft/pyright#1647]
**Describe the bug**
Pyright can't analyze the type of `numpy.pi` correctly.

**Environment**
* OS: Windows 10 20H2 64bit;
* VS Code: 1.54.3 with Pylance: v2021.3.2 or gvim 8.2 with coc-pyright 1.1.122;
* Python: 3.8.8 64bit;
* numpy: 1.19.5.
* 

**To Reproduce**
```python
import numpy as np

print(2 * np.pi)
```

Pyright detected the type of `np.pi` as `Module("numpy.testing") | Type[NoseTester]` but it should be `float`.

**Expected behavior**
Pyright should not detect the type of `numpy.pi` as `Module("numpy.testing") | Type[NoseTester]`.

**Screenshots or Code**

VS Code with Pylance:
![Snipaste_2021-03-18_13-59-40](https://user-images.githubusercontent.com/33576822/111580411-6fa74280-87f2-11eb-9bfa-169a2687dde5.png)

Gvim with coc-pyright:
![Snipaste_2021-03-18_14-00-42](https://user-images.githubusercontent.com/33576822/111580439-7b930480-87f2-11eb-9ce7-8265a2b5402d.png)

**Comment by erictraut** [comment]
Numpy 1.19 didn't have any type information, so without type stubs the type information will be highly inaccurate. If you want to use it for type checking, you should install type stubs or, even better, upgrade to numpy 1.20, which includes packaged type information.

Here's what I see with numpy 1.20.1 installed:

![Screen Shot 2021-03-17 at 11 28 40 PM](https://user-images.githubusercontent.com/7040122/111582762-8446f480-8778-11eb-87bc-01d0909df692.png)
